### ISSUE TITLE
Header X-Remote-User is not set 

### ISSUE THREAD

**Issue by timaebi** [clems4ever/authelia#65]
The example nginx configuration suggests that there is a `X-Forwarded-User` and `Remote-Groups` header sent to the server behind the proxy. However, this is not the case. There is no `X-Remote-User` and no `X-Remote-Groups` response header set in the answer coming from the `/verify` endpoint.
Also looking at the code, I couldn't find anything that would add them. It would be cool if they were set, such that a backend application is able to determine the the username. 

```
location = /secret.html {
            auth_request /auth_verify;
            
            auth_request_set $user $upstream_http_x_remote_user;
            proxy_set_header X-Forwarded-User $user;
            auth_request_set $groups $upstream_http_remote_groups;
            proxy_set_header Remote-Groups $groups;
            auth_request_set $expiry $upstream_http_remote_expiry;
            proxy_set_header Remote-Expiry $expiry;
        }
```

Btw:  http://scooterlabs.com/echo could be helpful as a backend to see all headers.

@clems4ever Thank you for your great work. This app is amazing.

**Comment by clems4ever** [comment]
This is clearly a mistake. Thanks for reporting. I think I would have need it for OAuth integration at some point.
Since a I have no access to my computer (put differently I'm in vacation ;) ), can you wait until beginning of September?

Thank you very much for your feedback on Authelia BTW,  this is very nice and encouraging. What are you using Authelia for?

**Comment by timaebi** [comment]
I quickly fixed it myself.
I currently using it for my self but in the future I may use it for my associations homepage.

**Comment by clems4ever** [comment]
Awesome! Do not hesitate to contribute to mainstream if you wish. Great you plan to use it for your association. Would you use U2F?

**Comment by clems4ever** [comment]
Review is done and waiting for a fix not to edit the documentation in the commit. Everything else is ok.

**Comment by clems4ever** [comment]
Closed with #92.
